test/parallel_api/algorithm/alg.sorting/stable_sort_by_key.pass.cpp

```diff
@@ -0,0 +1,26 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+#include "support/utils.h"
+
+#include "sort_by_key_common.h"
+
+int
+main()
+{
+    test_all_policies(StableSort{});
+    return TestUtils::done();

```

Comment:
if we skip some tests in some cases we should return an error/skip code, for example
return TestUtils::done(_ENABLE_RANGES_TESTING);

Reply:
It makes sense doing if there is a combination of macros when the test is skipped entirely. There is no such combination for this test.

Reply:
I think Dmitry is correct, we return a skip code if the test had not run at all. If some tests were run, it's `done()`